com.woltlab.wcf/templates/headIncludeJavaScript.tpl

```diff
@@ -8,6 +8,7 @@
 	var WSC_API_URL = '{@$__wcf->getActivePath()}';
 	var SECURITY_TOKEN = '{@SECURITY_TOKEN}';
 	var LANGUAGE_ID = {@$__wcf->getLanguage()->languageID};
+	var LANGUAGE_CODE = '{@$__wcf->getLanguage()->getFixedLanguageCode()}';
```

Comment:
I'm wondering if this can / should use `elBySel('html').getAttribute('lang')`. The attribute is already filled with the proper value.

Reply:
`document.documentElement.lang` is sufficient, because this is a global attribute. I agree, using the existing data is better than introducing another variable.